corehq/apps/hqcase/api/get_list.py

```diff
@@ -96,6 +96,22 @@ def get_list(domain, params):
     return ret
 
 
+def unpack_query_dict(query_dict):
+    """
+    Unpacks a ``QueryDict`` so that the values of duplicate keys are not
+    lost.
+
+    >>> qd = QueryDict('foo=one&bar=two&bar=three')
+    >>> unpack_query_dict(qd)
+    {'foo': 'one', 'bar': ['two', 'three']}
```

Comment:
I think there is a Django inbuilt way to get this by doing `request.GET.getlist('key')` that can be used here instead of implementing on own. You will just have to pass the `request.GET` to `get_list` instead of `request.GET.dict()`